fern/openapi-ir/definition/example.yml

```diff
@@ -0,0 +1,112 @@
+types:
+  SchemaInstanceId:
+    docs: |
+      Uniquely identifies an instance of a schema. For nested instances,
+      the id will be a dot-delimited path, e.g. foo.bar.baz.
+    type: string
+
+  Example:
+    docs: |
+      An OpenAPI example, which is either fully specified, partially
+      specified, or a reference to another schema instance's example.
+    union:
+      full: FullExample
+      partial: PartialExample
+      reference: ReferencedExample
+
+  FullOrPartialExample:
+    docs: |
+      Specifically designed for the PartialObjectExample.
+      The included properties can either be full or partial
+      examples, but not references.
+    union:
+      full: FullExample
+      partial: PartialExample
+
+  FullExample:
+    union:
+      primitive: PrimitiveExample
+      object: FullObjectExample
+      array: ArrayExample
+      map: MapExample
+      optional: FullExample
+      enum: EnumExample
+      reference: FullExample
+      literal: LiteralExample
+      oneOf: FullOneOfExample
+      nullable: FullExample
+      unknown: {}
+
+  PrimitiveExample:
+    union:
+      int: integer
+      int64: long
+      float: double
+      double: double
+      string: string
+      datetime: datetime
+      date: date
+      base64: base64
+      boolean: boolean
+
+  FullObjectExample:
+    properties:
+      properties: map<string, FullExample>
```

Comment:
would be down to inline as `map<string, FullExample>` on `FullExample`

Reply:
Sweet, I'll reuse the `PropertyKey` alias you defined for the union since it would apply well here, too.